### PR TITLE
Add a configuration module

### PR DESCRIPTION
**Pull request by beck**

### CODE REVIEW

**Comment by blaix** [comment]
Can you implement this entirely in yolapy instead? After discussion with @RayeN and @thomasw, we decided not to make yoconfig public.

**Comment by beck** [comment]
Added module `yolapy.configuration` with methods `configure` and `get_config` (sans extended django functionality)

File: tests/test_configuration.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from unittest import TestCase
+
+from mock import patch
+from yolapy import configuration
+from yolapy.configuration import configure, get_config
+
+
+@patch.dict('yolapy.configuration.config', clear=True)
```
Comment (blaix) [inline tests/test_configuration.py:8]: This is cool. I like this.

File: yolapy/configuration.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+config = {}
+_default = object()
+
+
+def configure(**kwargs):
+    """Save all keyword arguments as (key=value) configuration."""
+    config.update(kwargs)
+
+
+def get_config(key, default=_default):
```
Comment (blaix) [inline yolapy/configuration.py]: There's two different contexts for the concept of a default here which makes this a bit confusing for me. What about renaming `_default` to `_missing`?

**Comment by blaix** [comment]
I am :+1: on this pending [my comment about `_default`](https://github.com/yola/yolapy/pull/11#discussion_r40419385).

File: yolapy/configuration.py (hunk 2 above)
Comment (beck) [inline yolapy/configuration.py]: I like `_missing`

**Comment by beck** [comment]
updated

**Comment by blaix** [comment]
:+1:

**Pull request merged**